### ISSUE TITLE
Don't use spacebar to interact with boost/favorite buttons, only use for scrolling by page

### ISSUE THREAD

**Issue by vitriolix** [mastodon/mastodon#26824]
### Pitch

Currently you can use spacebar to scroll your feed, a very standard thing.  But, when you click on one of the action buttons on a post (star, boost, etc) the next time you hit spacebar it will unclick the button you just pressed instead of scrolling.

### Motivation

This is a minor but fundamental issue with the current UI and would really help browsing easily on desktop

**Comment by ClearlyClaire** [comment]
Hi, I think *not* having space activate the button would be surprising for everyone who relies on keyboard navigation.

**Comment by vitriolix** [comment]
You are right, accessibility is an important thing to consider.  I think the trouble I'm talking about won't have an impact there as it happens after you click on an action button that spacebar changes it's function. It does this without highlighting the button so you can't tell that this will happen, whereas if you tab to highlight a button spacebar behaves as expected as a toggle.

**Comment by diondiondion** [comment]
This is a bit tricky – the best solution to this particular issue would probably be to show a visible focus outline on buttons after they were clicked, but that's considered a bit distracting by many. We're using the fairly common `:focus-visible` selector for showing focus outlines on buttons (and I admit there's a lot of potential for improving how obvious this is in our web app).

That said, using "Space" to scroll is always going to be fairly "risky". I will close this as wontfix and would suggest using the <kbd>PageUp</kbd> or <kbd>PageDown</kbd> keys to manually scroll the page, since the functionality of those is consistent and doesn't change depending on which element is currently in focus.
